docs/source/contributing.rst

```diff
@@ -1,16 +1,19 @@
 .. _contributing:
 
-##########################
-Flytekit Contributor Guide
-##########################
+###########################
+Flytekit Contribution Guide
+###########################
 
 First off, thank you for thinking about contributing! Below you'll find instructions that will hopefully guide you through how to fix, improve, and extend Flytekit.
 
-Please also take some time to read through the design guides, which describe the various parts of Flytekit and should make contributing easier.
+Please also take some time to read through the `design guides <https://docs.flyte.org/projects/flytekit/en/latest/design/index.html>`__, which describe the various parts of Flytekit and should make contributing easier.
```

Comment:
can we use the sphinx link for this?
```
:std:ref:`design guides <design>`
```

i think it's that.

Reply:
Done!